src/org/usfirst/frc2175/subsystem/intake/DreamIntakeSubsystem.java

```diff
@@ -11,12 +12,41 @@
     private Talon dreamIntakeMainBeltTalon;
     private Talon dreamIntakeLiftTalon;
 
+    private double sideBeltSpeedForward;
+    private double sideBeltSpeedReverse;
+    private double mainBeltSpeedForward;
+    private double mainBeltSpeedReverse;
+
     public DreamIntakeSubsystem(RobotConfig robotConfig) {
         WiringConfig wiringConfig = robotConfig.getWiringConfig();
 
         dreamIntakeSideBeltTalon = wiringConfig.getDreamIntakeSideBeltTalon();
         dreamIntakeMainBeltTalon = wiringConfig.getDreamIntakeMainBeltTalon();
         dreamIntakeLiftTalon = wiringConfig.getDreamIntakeLiftTalon();
+
+        IntakeConfig intakeConfig = robotConfig.getIntakeConfig();
+
+        sideBeltSpeedForward = intakeConfig.getSideBeltSpeedForward();
+        sideBeltSpeedReverse = intakeConfig.getSideBeltSpeedReverse();
+        mainBeltSpeedForward = intakeConfig.getMainBeltSpeedForward();
+        mainBeltSpeedReverse = intakeConfig.getMainBeltSpeedReverse();
```

Comment:
And you don't need to pull these values out of `IntakeConfig` here. `DreamIntakeSubsystem` actually doesn't need to grab anything out of `IntakeConfig` right now.